src/GUI/MainForm.Designer.cs

```diff
@@ -970,7 +970,8 @@ private void InitializeComponent()
             this.toolZoomValue.AutoSize = false;
             this.toolZoomValue.Enabled = false;
             this.toolZoomValue.Name = "toolZoomValue";
-            this.toolZoomValue.Size = new System.Drawing.Size(36, 22);
+            this.toolZoomValue.Font = new System.Drawing.Font(System.Drawing.FontFamily.GenericMonospace, 8);
+            this.toolZoomValue.Size = new System.Drawing.Size(50, this.standardToolStrip.ClientSize.Height);
```

Comment:
And after 974 I would add an `else` case that if we are running under windows, we only set the size as before:
`
else
{            
    this.toolZoomValue.Size = new System.Drawing.Size(36, 22);
}`